src/index.js

```diff
@@ -1,54 +1,90 @@
 import './style.css';
 import Tasks from './modules/Tasks.js';
+import addHtml from './modules/AddHtml.js';
+import remoHtml from './modules/RemoHtml.js';
+import editHtml from './modules/EditHtml.js';
+
 // Initiate Tasks class
 const tasksCl = new Tasks();
 
-// Load example tasks to the class
-const exampleTasks = [
-  {
-    description: 'Complete a daily challenge',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Update and polish projects',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Respond e-mails',
-    completed: true,
-    index: 3,
-  },
-];
-
-exampleTasks.forEach((t) => {
-  tasksCl.newTask(t.description, t.completed, t.index);
-});
-
 const todoUl = document.getElementById('todo-ul');
 
-tasksCl.allTasks.forEach((task) => {
-  const liB = document.createElement('li');
-  liB.setAttribute('id', `id${task.taskIndex}`);
-  const x = document.createElement('INPUT');
-  x.setAttribute('type', 'checkbox');
-  liB.appendChild(x);
-  const p = document.createElement('p');
-  p.setAttribute('id', `p${task.taskIndex}`);
-  p.innerHTML = task.taskDesc;
-  liB.appendChild(p);
-
-  const i = document.createElement('i');
-  i.setAttribute('class', 'fa-solid fa-ellipsis-vertical');
-  liB.appendChild(i);
-
-  todoUl.appendChild(liB);
-});
+let currentIndex = 0;
+let numt = 0;
 
```

Comment:
- [ ] From project requirement by default, new tasks should have the property `completed` set to `false` you did well :+1: But the property `index` set to the value of the new array length (i.e. if I have 4th tasks in my list, the index of that task should equal to 4 ) in your case if my task is 4th the index equal 3 :cry:  Please fix it to follow the requirement :+1: 
- [ ] Another thing I would like to mention is as required all hardcoded items from the tasks array should be removed. But in your case that action is impossible . Please fix it :smile:

- [ ] When I check a task the completed property needs to change from `false `to `true ` But in your case not even when I refresh the page. Please fix it :smile: 

![Screenshot from 2022-08-12 21-15-20](https://user-images.githubusercontent.com/6615394/184447878-9d89eb3d-a54c-4df3-9b54-64aff2faf6f6.png)

Reply:
Hello, thanks for all the suggestions, I will take a look at them now.

One thing on the last one about the 'completed' properties, as I understand shouldn't that be for the next exercise ? If I'm not mistaken, this one is just for the add, remove and edit features

Reply:
@codecaiine 
> * Another thing I would like to mention is as required all hardcoded items from the tasks array should be removed. But in your case that action is impossible . Please fix it 😄

Sorry to bother, one more thing, I'm not sure about this point. I had some example items hardcoded previously which I removed, maybe I'm not understanding which parts are hard-coded and shouldn't be, could I ask for an example ? Thanks

Reply:
Hi @mullenlucas, I just noticed it ,  sorry you are right this is supposed to be for the following milestone :smile:

Reply:
> Hi @mullenlucas, I just noticed it, sorry you are right this is supposed to be for the following milestone 😄
> 
> Please make these changes below and submit the next reviewer will check 👍
> 
> > * But the property `index` set to the value of the new array length (i.e. if I have 4th task in my list, the index of that task should equal to 4 ) in your case if my task is 4th the index equal 3 cry  Please fix it to follow the requirement +1

Excellent, thank you

Reply:
> * [ ]  As I can see nothing works as required I can't add any new task when I write a task and press on the icon or enter nothing is added to the list kindly check that and work on it to be fixed 👌🏻💯💯
> 
>  Project.Title.-.Google.Chrome.2022-08-13.00-44-16.mp4

Hello @Reem-lab , thank you for your revision

However I'm a little lost, previous reviewers could run those functionalities. I'm not sure how to proceed, I ran multiple tests on different browsers (Mozilla, Chrome and Edge) and they all seem to work.
I add a video of a demonstration on it working:

https://user-images.githubusercontent.com/69157785/184459354-8baa2a40-0e0c-4c1c-9738-5cedb95be755.mp4

I am not sure what might be happening, as I understand it is working, don't really know what to modify. 
Please let me know any suggestion on how to proceed

Regards,
Lucas